docs/expressions.md

```diff
@@ -330,6 +345,20 @@ Remove any values lower than or equal to number from a series. Will error if thi
 
 Remove any NaN or Inf values from a series. Will error if this operation results in an empty series.
 
+## epoch()
+
+Returns the Unix epoch in seconds of the expression start time (scalar).
+
+## filter(series, number)
+
+Returns all results in series that are a subset of anything in number, or
+that have number as a subset. Useful with the limit and sort functions to
+return the top X results of a query.
+
+## limit(number, count)
```

Comment:
Then you shouldn't use number as the type for the second argument, as you can't have a subset of "A number is a single numeric value with a group associated with it.". We should add a new type called NumerSeries or NumberSet.